.drone.yml

```diff
@@ -896,4 +896,43 @@ volumes:
 trigger:
   event:
   - promote
+---
+kind: pipeline
+type: docker
+name: dispatch
+
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+  - name: dispatch
+    image: rancher/dapper:v0.5.2
```

Comment:
this is just a curl in a shell script, do we really need dapper? just use alpine or something small?

Reply:
if you did that you wouldnt need the volume mount or anything, just adds an extra container overhead thats unnecessary

Reply:
Dapper is already in scope for the build in drone. We can slim it down at a later date with some trial and error but if this has worked elsewhere we should be able to leverage it here.